packages/app/background-jobs-common/src/background-job-processor/processors/AbstractBackgroundJobProcessor.ts

```diff
@@ -121,20 +124,23 @@ export abstract class AbstractBackgroundJobProcessor<
   ) {
     this.config = config
```

Comment:
🟢 I think we always need to sanitize `redisConfig` (excluding `this.redis`, please check my prev comment) so maybe we can save it already sanitized to avoid having it in several places